src/Spotify/Album.java

```diff
@@ -0,0 +1,84 @@
+package Spotify;
+import java.io.*;
+import java.util.ArrayList;
+import Spotify.Library;
+import Spotify.Album;
+import Spotify.Playlist;
+
+public class Album extends Library {
+
+  // instance variables
+  private String artist;
+  private int tracks; 
+  //private ArrayList<Song> Library = new ArrayList<Song>();
+  private ArrayList<Song> Album = new ArrayList<Song>();
```

Comment:
'Album' is confusing as an instance variable name since it is the same name as the class and starts with an upper case letter (it shouldn't)